packages/calcite-components/src/components/action-pad/action-pad.scss

```diff
@@ -5,6 +5,7 @@
  *
  * @prop --calcite-action-pad-corner-radius: Specifies the component's border radius.
  * @prop --calcite-action-pad-expanded-max-width: Specifies the component's maximum width when `expanded` and the `layout` is `"vertical"`.
+ * @prop --calcite-action-pad-items-space" Specifies the space between slotted components in the component.
```

Comment:
typo:

`* @prop --calcite-action-pad-items-space: Specifies the space between slotted components in the component.`